Applications/PaintBrush/EraseTool.cpp

```diff
@@ -65,3 +65,9 @@ void EraseTool::on_contextmenu(GContextMenuEvent& event)
     m_context_menu->popup(event.screen_position());
 }
 
+Color EraseTool::get_color() {
```

Comment:
Missing break before the `{`.
Please use `clang-format` to automatically format code to fit the rest of the project.